README.md

```diff
@@ -3,82 +3,95 @@
 A process for collaborating on substantial changes to Concourse.
 
 
-## What should be proposed as an RFC?
+## Providing feedback on an RFC
 
-RFCs should be opened for changes that have a substantial impact on Concourse
-users and contributors. RFCs enable the community to collaborate during the
-architecture and feature design process, before getting to code and
-implementation.
+Everyone is encouraged to provide feedback on RFCs!
 
-An RFC may not be necessary for changes that are narrow enough in scope with
-limited impact to the rest of Concourse. If you feel that this is the case, you
-can cut straight to submitting a PR, though it's still a good idea to have an
-issue opened first to provide additional context. Do note however that pull
-requests and issues may be closed with a polite request to submit an RFC first.
+RFC review should be focused on bringing clarity to the proposal, resolving
+open questions and surfacing risks and drawbacks. The [Concourse design
+principles](DESIGN_PRINCIPLES.md) should be used to guide the solution and to
+determine whether the proposal should be accepted.
 
-If you're not sure whether to open an RFC for a change you'd like to propose,
-feel free to discuss beforehand in [Discord](https://discord.gg/MeRxXKW) - just
-ping `@rfc-czars` or guauge interest in `#contributors`.
+Questions and comments should be left via PR review on individual lines so that
+they can be marked as resolved. Top-level comments and pull-request reviews are
+OK for overarching commentary, but in general line-wise comments are preferred.
+
+Using GitHub reactions not only on the proposal itself but on other feedback
+comments is encouraged as a way to measure consensus without cluttering the
+comment thread.
 
 
-## Providing feedback to an RFC
+## What should be proposed as an RFC?
+
+RFCs should be opened for changes that have a substantial impact on Concourse
+users or contributors. RFCs enable the community to collaborate during the
+architecture and feature design process, providing clarity and validation
+before jumping to implementation.
 
-This process is centered around pull requests. Feedback and questions should be
-left as comments on specific lines of the pull request's proposal document, so
-that they can be marked as resolved. This is to avoid an ever-growing sequence
-of comments at the top level.
+An RFC may not be necessary for changes that are narrow enough in scope and
+impact. If you feel that this is the case, you can jump straight to submitting
+a PR. Do note however that pull requests may be closed with a polite request to
+submit an RFC if we find that it needs broader feedback.
 
-Top-level comments and pull-request reviews are allowed for overarching
-commentary, but in general line-wise comments are preferred.
+If you're not sure whether to open an RFC for a change you'd like to propose,
+feel free to [ask in `#dev`](https://discord.gg/MeRxXKW).
```

Comment:
Unrelated to this PR, but access to `#dev` requires being a contributor - is there anything on Discord that indicates how to become a contributor? Should we add something to the channel description or is there a better way?

Reply:
The `#welcome` channel does but it's out of date (just says ask for it) - we can add a link to https://github.com/concourse/governance in the topic.